python/pyspark/sql/dataframe.py

```diff
@@ -3540,6 +3546,9 @@ def melt(
 
         .. versionadded:: 3.4.0
 
+        .. versionchanged:: 3.4.0
```

Comment:
3.5.0？

Reply:
I double checked that `melt` in connect was added in https://github.com/apache/spark/commit/3212fa960169b1f1c29d63185aa96d535798fcc4, since 3.4.0